packages/taiko-client/cmd/flags/driver.go

```diff
@@ -72,6 +72,15 @@ var (
 		Category: driverCategory,
 		EnvVars:  []string{"TAIKO_WRAPPER"},
 	}
+	// preconfirmation transition timestamp (unix seconds). Before this timestamp, old APIs + block gossip are used.
+	// At or after this timestamp, new commitment-based API and logic are enabled and old APIs are disabled.
+	PreconfTransitionTimestamp = &cli.Uint64Flag{
```

Comment:
unsure of whether itll be a block number or a timestamp right now but we can leave this and confer with the partners

Reply:
I though it will be time-based since we moved Shasta to time based one